### PR TITLE
feat(release): Edit Release - Create tab Clearing Details

### PR DESCRIPTION
**Pull request by tuannn2**
### Edit Release

- Handle Case Release have Commercial Details
- Define Object , State Clearing Information
- Create Tab Clearing Details

Clearing_detail_PR5

### CODE REVIEW

File: src/app/[locale]/components/editRelease/[id]/releases/ClearingDetail/ClearingDetails.module.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+.button-plain:hover {
+    border-color: #dbdce4 !important;
+    color: black !important;
+    background-color: #f8f9fa !important;
+}
+
+.header {
+    background-color: #5D8EA9;
+    color: white;
+}
```
Comment (heliocastro) [inline src/app/[locale]/components/editRelease/[id]/releases/ClearingDetail/ClearingDetails.module.css:45]: Nope nope nope in several levels.
We need consistency. Please don't add another internal CSS

**Review by heliocastro — CHANGES_REQUESTED** [review]
No internal CSS
You are touching things that are global, button, sidebar and modifying locally.
ONLY itens that is not global, should be in the custom CSS.
Please use global CSS for thins that are generic

File: src/app/[locale]/components/editRelease/[id]/releases/ClearingDetail/ClearingDetails.module.css (hunk 1 above)
Comment (tuannn2) [reply src/app/[locale]/components/editRelease/[id]/releases/ClearingDetail/ClearingDetails.module.css]: Thank you for your comment, @heliocastro  
Fixed.

**Comment by hoangnt2** [comment]
Rename **releases** folder to **components**. component is react component not sw360 component

**Comment by tuannn2** [comment]
> Rename **releases** folder to **components**. component is react component not sw360 component

Done.

**Review by heliocastro — APPROVED** [review]

**Pull request merged**